tests/unittest_scoped_nodes.py

```diff
@@ -1078,15 +1078,16 @@ def g1(x):
                 print(x)
 
             @f(a=2,
-               b=3)
+               b=3,
+            )
             def g2():
                 pass
         """
         astroid = builder.parse(data)
         self.assertEqual(astroid["g1"].fromlineno, 4)
         self.assertEqual(astroid["g1"].tolineno, 5)
-        self.assertEqual(astroid["g2"].fromlineno, 9)
-        self.assertEqual(astroid["g2"].tolineno, 10)
+        self.assertEqual(astroid["g2"].fromlineno, 10)
+        self.assertEqual(astroid["g2"].tolineno, 11)
```

Comment:
Is this change necessary?

Reply:
Yes, this is a reproducer for #1350. I guess I'd have to branch here as well, because on <3.8 the range reported by astroid is (9, 10).

Reply:
Ah sorry, my bad!